packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/face-value/index.hbs

```diff
@@ -1,19 +1,76 @@
 <ActionCardContainer
+  class="face-value-card"
   @header="Prepaid card funding"
   @isComplete={{@isComplete}}
-  data-test-layout-customization-is-complete={{@isComplete}}
 >
+  <ActionCardContainer::Section @title="Choose the face value of your prepaid card" class="face-value-card__section">
+    <div class="face-value-card__container">
+      <p class="face-value-card__exchange">
+        <span>Prepaid cards are denominated in SPEND:</span>
+        <span class="face-value-card__exchange-rate">
+          {{svg-jar "spend" width="20" height="20" role="presentation"}}
+          §1 SPEND = ${{this.spendToUSDRate}} USD
+        </span>
+      </p>
+      <CardPay::LabeledValue @vertical={{true}} @label="Funding From:">
+        <CardPay::IssuePrepaidCardWorkflow::FaceValue::BalanceView
+          @walletAddress={{this.layer2Network.walletInfo.firstAddress}}
+          @depotAddress={{this.layer2Network.depotSafe.address}}
+          @token={{this.fundingToken}}
+          @balance={{format-token-amount this.fundingToken.balance}}
+        />
+      </CardPay::LabeledValue>
+      {{#if @isComplete}}
+        <CardPay::LabeledValue @vertical={{true}} @label="Face Value:">
+          <CardPay::BalanceDisplay
+            class="face-value-card__selected-amount-display"
+            @size="large"
+            @icon={{this.faceValueToken.icon}}
+            @balance={{this.selectedFaceValue}}
+            @usdBalance={{mult this.selectedFaceValue this.spendToUSDRate}}
+            @symbol={{this.faceValueToken.symbol}}
+            @sign={{this.faceValueToken.sign}}
+            @text={{concat @amountInFundingToken " " this.fundingToken.symbol "*"}}
+            data-test-face-value-display
+          />
+        </CardPay::LabeledValue>
+      {{else}}
+        <div class="face-value-card__options">
+          {{#each this.amountOptions as |amount|}}
+            <CardPay::IssuePrepaidCardWorkflow::FaceValue::Option
+              @checked={{eq amount this.selectedFaceValue}}
+              @amount={{amount}}
+              @token={{this.faceValueToken}}
+              @amountInUSD={{mult amount this.spendToUSDRate}}
+              @amountInFundingToken={{@amountInFundingToken}}
+              @fundingTokenSymbol={{this.fundingToken.symbol}}
+              @onInput={{fn this.chooseFaceValue amount}}
+            />
+          {{/each}}
```

Comment:
This is more a larger design question; I found issue 847 related to having commas for larger values, should that maybe be applied globally? I think it can help make it easier to recognise the values. Here it’s helped because the bottom values are an order of magnitude larger than the top ones, but I feel like in general it’s preferable.

![image](https://user-images.githubusercontent.com/43280/123148257-a0b49600-d425-11eb-845a-d8f59576e74b.png)

Reply:
Yes, we want the commas wherever there's a large enough value. I thought that ticket was already done, but I'm not seeing the commas anywhere. Maybe some sort of regression happened, and it's not being applied to the helpers.

<img width="768" alt="no-commas" src="https://user-images.githubusercontent.com/16160806/123282241-52ac9a80-d4d8-11eb-9ce2-5530fbda5b26.png">